secure-boot-guide.md

```diff
@@ -0,0 +1,73 @@
+## Build td-shim

```

Comment:
Need one line to describe the high level feature.

And add link for [secure_boot.md].

Also, please add [secure_boot.md] to [td-shim spec.md]